src/cljs/rems/administration/create_workflow.cljs

```diff
@@ -9,54 +10,99 @@
             [rems.collapsible :as collapsible]
             [rems.spinner :as spinner]
             [rems.text :refer [text]]
-            [rems.util :refer [dispatch! fetch post!]]))
+            [rems.util :refer [dispatch! fetch parse-int post! put!]]))
 
 (rf/reg-event-fx
  ::enter-page
- (fn [{:keys [db]}]
-   {:db (assoc db
-               ::form {:type :dynamic}
-               ::loading? true)
-    ::fetch-actors nil}))
-
-(rf/reg-sub ::loading? (fn [db _] (::loading? db)))
+ (fn [{:keys [db]} [_ workflow-id]]
+   (let [parsed-id (parse-int workflow-id)]
```

Comment:
Maybe the parsing could be already in `spa.cljs` and this function could be purer.

Reply:
good idea, done